docs/how_to/use_estimator.rst

```diff
@@ -0,0 +1,140 @@
+###########################
+Use the Estimator primitive
+###########################
+
```

Comment:
For the two primitives howtos (I will just comment in this one) we may like to note that the outcome, by default is ideal. The Sampler is the ideal distribution and the Estimator is the expectation value off an ideal distribution. Both can be configured to do sampling (shots). I know these are about the Terra primitives but for instance the Aer primitives do sampling by default and switching the same code to simply use say and Aer Estimator in VQE were it was working with the Terra one may fail give the same answer. I know this is not an aspect of this howto as such, but I am wondering if its worth to have some note here.